engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorFactoryImpl.java

```diff
@@ -702,8 +707,16 @@ public String toString() {
 		 */
 		private final boolean traversableResolverResultCacheEnabled;
 
+		/**
+		 * The constraint validator payload.
+		 */
 		private final Object constraintValidatorPayload;
 
+		/**
+		 * The constraint validator initialization context.
+		 */
+		private final HibernateConstraintValidatorInitializationContextImpl constraintValidatorInitializationContext;
```

Comment:
So with this `HibernateConstraintValidatorInitializationContext` added here I suppose we could use it in `ValidatorImpl` [here](https://github.com/hibernate/hibernate-validator/blob/master/engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorImpl.java#L148). instead of creating one more context we could just
```java
this.constraintValidatorInitializationContext = validatorFactoryScopedContext.getConstraintValidatorInitializationContext();
```

Reply:
Yeah, that was the whole point of my commit in fact :). I missed a step.